NewSpring/ApollosProject
# Fix Prayer Menu

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR fixes the prayer menu so that it displays and functions correctly on both iOS and Android.

### How do I test this PR?
Run it locally on both iOS and Android and see that the prayer menu displays and that you can move between tab items.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [n/a] Closes [DEV-XXX](#url)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
